client/components/backup-retention-management/index.tsx

```diff
@@ -234,8 +235,12 @@ const BackupRetentionManagement: FunctionComponent< OwnProps > = ( {
 			updateRetentionRequestStatus === BACKUP_RETENTION_UPDATE_REQUEST.FAILED
 		) {
 			setConfirmationDialogVisible( false );
+			dispatch( {
+				type: JETPACK_BACKUP_RETENTION_UPDATE_RESET,
```

Comment:
This is added to prevent auto-redirecting from the cancellation page to the settings page. For example, when the retention is updated to (7, 30, 120, or 365) from the settings page and if we go to the cancel page, without this change it will redirect back to the settings page since the internal state of `updateRetentionRequestStatus` is not reset and is still set to `SUCCESS`, we need to reset it to the initial state `UNSUBMITTED` to prevent auto-redirection.